db/index.js

```diff
@@ -9,6 +9,7 @@ db.once('open', () => {
 });
 
 const imageSchema = new mongoose.Schema({
+  restaurantId: Number,
   url: String,
   name: String,
   type: String,
```

Comment:
what is type? not very descriptive

Reply:
@ez-li Type is either from the restaurant, OpenTableDiner, or Foodspotting
Maybe rename to source?